docs/dm/zh_CN/maintenance/dm-ansible.md

```diff
@@ -340,21 +340,25 @@ $ ansible-playbook stop.yml
 ### 升级组件版本
 
 #### 下载 DM binary
+1. 以 `tidb` 用户登录中控机并进入 `/home/tidb/` 目录，备份原有的 dm-ansible 目录，删除旧版本压缩文件：
+    ```
+    $ mv dm-ansible dm-ansible-bak
+    $ rm -rf dm-ansible-latest.tar.gz
+    ```
 
-1. 删除原有的 downloads 目录下文件。
+2. 下载最新 DM-Ansible，并且根据 dm-ansible-bak 中的已有配置更新 inventory.ini 等文件。
 
-```
-$ cd /home/tidb/dm-ansible
-$ rm -rf downloads
-```
+    ```
+    $ wget http://download.pingcap.org/dm-ansible-latest.tar.gz
+    $ tar -xzvf dm-ansible-latest.tar.gz
+    $ mv dm-ansible-latest dm-ansible
+    ```
 
-2. 使用 playbook 下载最新的（latest） DM binary，自动替换 `/home/tidb/dm-ansible/resource/bin/` 目录下 binary 文件。
-```
-$ ansible-playbook local_prepare.yml
-```
+3. 使用 playbook 下载最新的（latest） DM binary。
 
-> 如果需要根据 Git Commit Hash 下载指定版本的 DM binary，可以直接从 http://pingcap-dev.hk.ufileos.com/builds/pingcap/tidb-enterprise-tools/{git-commit-hash}/centos7/dm-linux-amd64.tar.gz 进行下载。
```

Comment:
should we need to support download specified version of DM through DM-Ansible?

Reply:
ping @liubo0127 we have `v1.0 alpha` now